### PR TITLE
Added sponsor, current team & ongoing projects.

### PR DESCRIPTION
**Pull request by Spnetic-5**
Test link: https://sra-vjti.vercel.app/

### CODE REVIEW

File: components/Activities/Activities.module.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
     padding: 2em;
     display: flex;
     flex-direction: column;
-    justify-content: start;
+    justify-content: flex-start;
```
Comment (kkothari2001) [inline components/Activities/Activities.module.scss:38]: What is the reason for this change?

File: data/notification.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ const NotificationData = [
   'March 21, 2021 - Seminar for First Years',
   'April 11, 2021 - Wall-E Worshop',
   'Mario Worshop for First Years',
+  'Eklavya Mentorship Program 2021'
```
Comment (kkothari2001) [inline data/notification.js:5]: Pls remove all previous notifs and only keep Eklavya

**Review by kkothari2001 — CHANGES_REQUESTED** [review]
Thank you @Spnetic-5  for this feature! Everything looks good, and only one very very minor change.

File: components/Activities/Activities.module.scss (hunk 1 above)
Comment (Spnetic-5) [reply components/Activities/Activities.module.scss:38]: This warning: `start value has mixed support, consider using flex-start instead`

**Comment by kkothari2001** [comment]
Thanks @Spnetic-5 !
